share/bin/info.py

```diff
@@ -0,0 +1,25 @@
+from share.models import SourceConfig
+from share.bin.util import command
+
+
+@command('List SourceConfigs')
+def sources(args, argv):
+    """
+    Usage: {0} sources [-e | -d]
+
+    Options:
+        -e, --enabled        Only list enabled SourceConfigs
+        -d, --disabled       Only list disabled SourceConfigs
+
+    Print out a list of currently installed source configs
+    """
+    configs = SourceConfig.objects.all()
+
+    if args['--enabled']:
+        configs = configs.filter(enabled=True, source__is_deleted=False)
```

Comment:
Should be `disabled=False`, because I guess we thought that name made sense at the time.